JavaScript/tests/syntax_test_js_bindings.js

```diff
@@ -0,0 +1,38 @@
+// SYNTAX TEST "Packages/JavaScript/JavaScript.sublime-syntax"
+
+const x = value;
+//    ^ meta.binding.name variable.other.readwrite
+
+
+const [ x, y, ...z, ] = value;
+//    ^^^^^^^^^^^ meta.binding.destructuring.sequence
+//      ^ meta.binding.name variable.other.readwrite
+//       ^ punctuation.separator.comma
+//         ^ meta.binding.name variable.other.readwrite
+//          ^ punctuation.separator.comma
+//            ^^^ keyword.operator.spread
+//               ^ meta.binding.name variable.other.readwrite
+//                ^ punctuation.separator.comma
+
+const [ x, [a, b], z] = value;
+//    ^^^^^^^^^^^^^^^ meta.binding.destructuring.sequence
+//         ^^^^^^ meta.binding.destructuring.sequence meta.binding.destructuring.sequence
```

Comment:
This should probably be `meta.binding.destructuring.sequence meta.binding.destructoring.sequence` to ensure it is doubled up, right?

Reply:
Is there a typo? I'm not seeing it.

Reply:
Sorry, it must have been wrapped on my screen last night and I didn't notice the second scope name.